addons/pinniped/config-controller/README.md

```diff
@@ -15,4 +15,18 @@ $ ./hack/test.sh
 To run static checks (e.g., `go fmt`, `go vet`, `golangci-lint`) and the tests:
 ```sh
 $ ./hack/check.sh
-```
\ No newline at end of file
+```
+
+To generate the default pinniped addon secret:
+
+```sh 
+# manually via ytt
+ytt -f ./hack/ytt/default-package-secret.yaml -f ./hack/ytt/values.yaml -v tkr=v1.23.3---vmware.1-tkg.1 -v iaas=vsphere
```

Comment:
in `generate-package-secret.sh`, we also list `-f schema.yaml`...do we need to list that here?

similar to another comment, is it worth having this line here in addition to the `./hack/generate-package-secret.sh`? we will have to remember to keep this `ytt` line up to date

Reply:
I'll update!